libs/common/src/helpers/currency.ts

```diff
@@ -1,18 +1,14 @@
 import { CurrencyAndAmount, CurrencyAndAmountString } from '../types';
+import { getFormattedMonetaryValue } from './monetary-value';
 
 /**
- * Gets the formatted currency and amount using the 'en-GB' number formatting
- * for the amount
+ * Gets the formatted currency and amount with the amount formatted to 2 decimal
+ * places with thousands separators
  * @param currencyAndAmount - The currency and amount object
  * @returns The formatted currency and amount
  * @example
  * const amount = getFormattedCurrencyAndAmount({ currency: 'GBP', amount: 3.14159 }); // 'GBP 3.14'
  */
 export const getFormattedCurrencyAndAmount = (currencyAndAmount: CurrencyAndAmount): CurrencyAndAmountString => {
-  const formatter = new Intl.NumberFormat('en-GB', {
-    minimumFractionDigits: 2,
-    maximumFractionDigits: 2,
-  });
-  const formattedAmount = formatter.format(currencyAndAmount.amount);
-  return `${currencyAndAmount.currency} ${formattedAmount}`;
+  return `${currencyAndAmount.currency} ${getFormattedMonetaryValue(currencyAndAmount.amount)}`;
```

Comment:
`=> return `${currencyAndAmount.currency} ${getFormattedMonetaryValue(currencyAndAmount.amount)}`;` as no other logic is being used